test/internal/modules/billing/controller.js

```diff
@@ -300,4 +300,38 @@ experiment('internal/modules/billing/controller', () => {
       expect(view).to.equal('nunjucks/billing/batch-list');
     });
   });
+
+  experiment('.getBillingBatchInvoice', () => {
+    beforeEach(async () => {
+      request = {
+        params: {
+          batchId: 'test-batch-id'
+        }
+      };
+
+      await controller.getBillingBatchInvoice(request, h);
+    });
+
+    test('passes the required invoice data to the view', async () => {
+      const [, context] = h.view.lastCall.args;
+      const { invoice } = context;
+      console.log(invoice);
```

Comment:
Forgotten console.log here

Reply:
Noob error :-) Ok this is outdated now. I think James spotted this in the other ticket I worked on so this was fixed with a different PR.